libs/coin-modules/coin-solana/src/api/chain/index.ts

```diff
@@ -16,6 +16,8 @@ import { getEnv } from "@ledgerhq/live-env";
 import { Awaited } from "../../logic";
 import { NetworkError } from "@ledgerhq/errors";
 
+export const LATEST_BLOCKHASH_MOCK = "EEbZs6DmDyDjucyYbo3LwVJU7pQYuVopYcYTSEZXskW3";
```

Comment:
why is it here ? Is it a random hash ?

Reply:
I don't know its purpose.
I move it for convient reason, as it is used within api calls.